fe/fe-core/src/main/java/com/starrocks/http/BaseResponse.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Set;
 
 public class BaseResponse {
+    public static final String HEADER_QUERY_ID = "X-StarRocks-Query-Id";
```

Comment:
no need to put query id in header, can we put it in the response body?

Reply:
I forgot to delete query id. do we need exposing query id to user?